### PR TITLE
Various enhancements

### PR DESCRIPTION
**Pull request by adamchester**
1. renamed folders
2. make bash scripts executable
3. make sample a console executable

### CODE REVIEW

File: sample/Kmd.Logic.Sms.Client.Sample/Kmd.Logic.Sms.Client.Sample.csproj [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
+    <OutputType>Exe</OutputType>
```
Comment (paimaam) [inline sample/Kmd.Logic.Sms.Client.Sample/Kmd.Logic.Sms.Client.Sample.csproj]: why exe ?

File: sample/Kmd.Logic.Sms.Client.Sample/Program.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Kmd.Logic.Sms.Client.Sample
+{
+    public static class Program
+    {
+        public static int Main()
+        {
+            Console.WriteLine("Hello, world!");
```
Comment (paimaam) [inline sample/Kmd.Logic.Sms.Client.Sample/Program.cs]: can we talk among ourselves ;)

File: sample/Kmd.Logic.Sms.Client.Sample/Kmd.Logic.Sms.Client.Sample.csproj (hunk 1 above)
Comment (adamchester) [reply sample/Kmd.Logic.Sms.Client.Sample/Kmd.Logic.Sms.Client.Sample.csproj]: Our customers want to actually run the sample

File: sample/Kmd.Logic.Sms.Client.Sample/Program.cs (hunk 2 above)
Comment (adamchester) [reply sample/Kmd.Logic.Sms.Client.Sample/Program.cs]: No.

**Comment by paimaam** [comment]
Thank you for your contribution @adamchester  :)

**Pull request merged**